src/components/routes/Booking/BookingOptions/BookingOptionsCrypto/BookingOptionsCrypto.tsx

```diff
@@ -104,6 +109,9 @@ const BookingOptionsCrypto = ({ booking, currency, fromBee, history }: Props) =>
                       </div>
                     </div>
                   ))}
+                {!isTwoDaysFromNow && <div className="booking-options-error">
+                  <p>Bookings paid using Ethereum or ERC-20 tokens must be made at least two days in advance.</p>
```

Comment:
Today (2/12), I chose dates 2/14 to 2/15 and the message shows up.

If it says at least 2 days in advance, one might think those dates are technically 2 days in advance. Maybe it can be "more than two days in advance"?